mkt/ratings/models.py

```diff
@@ -59,6 +82,16 @@ class Meta:
     def get_url_path(self):
         return '/app/%s/ratings/%s' % (self.addon.app_slug, self.id)
 
+    def delete(self):
+        self.update(deleted=True)
+        # This should happen in the `post_save` hook.
+        # self.refresh(update_denorm=True)
+
+    def undelete(self):
+        self.update(deleted=False)
+        # This should happen in the `post_save` hook.
+        # self.refresh(update_denorm=True)
```

Comment:
The comment means the following line occurs in post_save.  (I guessed and then confirmed with Kris).  It seems redundant to me so I can just remove the comments.